actors/user/src/main/java/org/sunbird/user/actors/UserManagementActor.java

```diff
@@ -651,6 +655,41 @@ private void processUserRequest(
     TelemetryUtil.telemetryProcessingCall(userMap, targetObject, correlatedObject);
   }
 
+  private void userBoolFlagToNum(Map<String, Object> requestMap, Map<String, Object> userBooleanMap) {
```

Comment:
* userBoolFlagToNum to userFlagsToNum
*  this method will return calculated int value , caller need to use it.
* in request userBooleanMap Map<String,Object> can be chnage to <Stirng,Boolean>

Reply:
Done